packages/app/src/components/Root/Root.tsx

```diff
@@ -88,8 +106,10 @@ const Root: FC<{}> = ({ children }) => (
       <SidebarItem icon={AccountTreeIcon} to="/inventory" text="Inventory" />
       <SidebarItem icon={AccountCircle} to="/login" text="Login" />
       <SidebarDivider />
+      <SidebarIntro />
```

Comment:
Since this functionality is not yet implemented I think we should not show the intro boxes. Thinking that will be confusing for people.